client/my-sites/stats/stats-subscribers/index.tsx

```diff
@@ -65,10 +66,14 @@ const StatsSubscribersPage = ( { period }: StatsSubscribersPageProps ) => {
 		'subscribers-page'
 	);
 
+	// TODO: Pass subscribersTotals as props to SubscribersHighlightSection to avoid duplicate queries.
```

Comment:
Question: I thought react-query will handle all that, wouldn't it?

Reply:
Yes, you are correct since they have precisely the same params.